### PR TITLE
fix: Pool boosted UI

### PR DESCRIPTION
**Pull request by ChefMomota**
### Summary
🔄🔗🚀

Added `chainId` to query key for fetching boosted pool APRs. This allows the query to update the APRs based on the network the user is connected to.

> _`useQuery` changes_
> _`chainId` affects the APR_
> _Pools shift with seasons_

### Walkthrough
*  Add `chainId` parameter to query key of `useQuery` hook to refetch boosted pool APR when network changes ([link](https://github.com/pancakeswap/pancake-frontend/pull/8422/files?diff=unified&w=0#diff-fecd21c45f9d393643358542fcc35dc2f73d5edd2055bc61ee67c7e84aaff089L15-R15))

### CODE REVIEW

File: apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const useBoostedPoolApr = ({ contractAddress, chainId }: UseBoostedPoolAp
   const client = getViemClients({ chainId })
 
   const { data } = useQuery(
-    ['boostedPoolsApr'],
+    ['boostedPoolsApr', chainId],
```
Comment (chefjackson) [inline apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts:15]: `contractAddress` should also be a dependency as well?

File: apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ export const useBoostedPoolApr = ({ contractAddress, chainId }: UseBoostedPoolAp
   const client = getViemClients({ chainId })
 
   const { data } = useQuery(
-    ['boostedPoolsApr'],
+    ['boostedPoolsApr', client, contractAddress, chainId],
```
Comment (chefjackson) [inline apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts:15]: can remove client? As it's bound with chainId

Comment (ChefMomota) [reply apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts]: Done

**Review by chefjackson — APPROVED** [review]

File: apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts (hunk 2 above)
Comment (Siloshah) [reply apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts]: Done

**Pull request merged**